testing/unit_testing/sort_cohorts_test/test_SortCohorts.pf

```diff
@@ -0,0 +1,425 @@
+module test_SortCohorts
```

Comment:
Again, this is an excellent set of tests which adds confidence to this important routine.